train/helpers.py

```diff
@@ -183,3 +183,48 @@ def visualize_data(dataloader, model=None):
     if model:
         # show model prediction
         pass
+    
+    def nms(boxes, overlap_threshold=0.3):
+        """
+        boxes: bouning boxes coordinates, ie, tuple of 4 integers
+        overlap threshold: the threshold for which the overlapping images will be suppressed
+        return the coordinates of the correct bounding boxes
+        """
+        if len(boxes) == 0:
+            return []
+
+        pick = []
+
+        x1 = boxes[:, 0]
+        y1 = boxes[:, 1]
+        x2 = boxes[:, 2]
+        y2 = boxes[:, 3]
+
+        area = (x2 - x1 + 1) * (y2 - y1 + 1)
+        idxs = np.argsort(y2)
+
+        while len(idxs) > 0:
```

Comment:
perhaps some refactoring and comments on what some variables mean, or the algorithm idea